NumRush.py

```diff
@@ -223,8 +224,8 @@ def run(self):
     	
         global XO1,XO2,DISPLAYSURF, WHITE, BLACK, DarkColor, LightColor, DarkColor, RED, GREY, fontObj, megaFont, ResX, ResY, FPS, fpsClock
         XO1,XO2 = profile.get_color().to_string().split(',')
```

Comment:
I know this wasn't your but could you make this a bit more readable by sticking to using constants as globals so all these would be constants, you can define them directly above the function seeing as that's where they were used.

Reply:
It would probably make the declaration redundant if I defined them in multiple functions.

Reply:
The idea isn't to define them in multiple functions, the idea is to remove them from one function seeing as they're globals.
It'll help make the code here a bit cleaner and avoid having that long line as you can define them right above the class they're used. @srevinsaju @quozl any thoughts?

Reply:
I think it is readable, but splitting the line would be good.  flake8 asserts `E501 line too long (141 > 79 characters)`  Some of the names do not need to be globals, such as `XO1`.  Some do.  I would not use camel case, but that horse has bolted for this file, no point going back.  DarkColor is assigned twice.  Overall my impression is code that didn't get an effective peer review.  If you're willing to work it, go ahead.  But I'd split the porting patches out from the make it better patches, and wouldn't do any make it better patches until the porting patches had demonstrated good test results.